weblab/templates/base.html

```diff
@@ -29,7 +29,7 @@
     </div>
 
     <footer>
-      Cardiac Electrophysiology Web Lab &copy; 2013-{% now 'Y' %} University of Oxford, 2016-{% now 'Y' %} University College London
+      Cardiac Electrophysiology Web Lab &copy; 2013-{% now 'Y' %} University of Oxford, 2016-{% now 'Y' %} University College London, and 2016-{% now 'Y' %} University of Nottingham
```

Comment:
We could perhaps stop the Oxford part at 2021 as no developer is based there now?